src/Http/Controllers/Character/CharacterController.php

```diff
@@ -82,6 +83,7 @@ public function getCharactersData(Request $request)
                 return view('web::character.partials.delete', compact('row'))
                     ->render();
             })
+            ->escapeColumns([])
```

Comment:
This has me very nervous. Can we not just use `rawColumns()` where necessary (like those with blade templates as the escaping happens there) instead of just disabling it for everything please?